apps/balancer-tools/src/app/apr/round/(components)/TokenFilterInput.tsx

```diff
@@ -0,0 +1,49 @@
+"use client";
+
+import { useRouter, useSearchParams } from "next/navigation";
+
+import { POOLS_WITH_LIVE_GAUGES } from "#/lib/balancer/gauges";
+
+import { MultiSelectDropdown } from "./MultiSelectDropdown";
+
+const AVALIABLE_TOKENS = [
+  ...new Set(
+    POOLS_WITH_LIVE_GAUGES.flatMap((pool) =>
+      pool.tokens.map((token) => token.symbol),
+    ),
+  ),
+];
+
+export function TokenFilterInput() {
+  const router = useRouter();
+  const searchParams = useSearchParams();
+
+  const handleTokenSelect = (selectedItems: string[]) => {
+    const current = new URLSearchParams(Array.from(searchParams.entries()));
+
+    if (!selectedItems.length) {
+      current.delete("tokens");
+    } else {
+      current.set("tokens", selectedItems.join(","));
+    }
+
+    const search = current.toString();
+    const query = search ? `?${search}` : "";
+    router.push(window.location.pathname + query, { scroll: false });
+  };
+
```

Comment:
why not define this within `MultiSelectDropdown`?

Reply:
I'm moving `MultiSelectDropdown` to component, I created `MultiSelectDropdown` with the intent for it to be a reusable multi-select